firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java

```diff
@@ -24,6 +24,14 @@
 /** Settings used to configure a FirebaseFirestore instance. */
 @PublicApi
 public final class FirebaseFirestoreSettings {
+  /**
+   * Set this constant as the value for {@link
```

Comment:
Nit: Maybe `Constant to use with `, since we want to describe what the value is, rather than what someone can do with it.

Reply:
Sure